cmd/createFgaModel.go

```diff
@@ -0,0 +1,63 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+
+	"github.com/canonical/identity-platform-admin-ui/internal/authorization"
+	"github.com/canonical/identity-platform-admin-ui/internal/logging"
+	"github.com/canonical/identity-platform-admin-ui/internal/monitoring"
+	fga "github.com/canonical/identity-platform-admin-ui/internal/openfga"
+	"github.com/canonical/identity-platform-admin-ui/internal/tracing"
+	"github.com/spf13/cobra"
+)
+
+// createFgaModelCmd represents the createFgaModel command
+var createFgaModelCmd = &cobra.Command{
+	Use:   "create-fga-model",
+	Short: "Creates an openfga model",
+	Long:  `Creates an openfga model`,
+	Run: func(cmd *cobra.Command, args []string) {
+		apiUrl, _ := cmd.Flags().GetString("fga-api-url")
+		apiToken, _ := cmd.Flags().GetString("fga-api-token")
+		storeId, _ := cmd.Flags().GetString("store-id")
+		createModel(apiUrl, apiToken, storeId)
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(createFgaModelCmd)
+
+	createFgaModelCmd.PersistentFlags().String("fga-api-url", "", "The openfga API URL")
+	createFgaModelCmd.PersistentFlags().String("fga-api-token", "", "The openfga API token")
+	createFgaModelCmd.PersistentFlags().String("fga-store-id", "", "The openfga store to create the model in")
+	createFgaModelCmd.MarkPersistentFlagRequired("fga-api-url")
+	createFgaModelCmd.MarkPersistentFlagRequired("fga-api-token")
+	createFgaModelCmd.MarkPersistentFlagRequired("fga-store-id")
+}
+
+func createModel(apiUrl, apiToken, storeId string) {
+	logger := logging.NewNoopLogger()
```

Comment:
remind me again, why don't we want the normal logger?

Reply:
When I run an one-off cli command I don't expect to see the timestamp and all the extra info the logger provides. 

I can change it if you want, just felt counterintuitive.

Reply:
this would be called by a juju action or init container, still valuable to get the logs into loki imho

Reply:
This would mean that you would configure this write the command output to the file that loki reads from or that you would configure loki to read the stdout form the init container, so I would need to parse the env vars or add a config option for piping the output to a log file.

Reply:
as discussed, leave it be, we can always add it later